tests/spcs/test_registry.py

```diff
@@ -1,10 +1,13 @@
 import json
+import subprocess
```

Comment:
Do we use it somewhere?

Reply:
I only use subprocess.CalledProcessError for some tests, I've changed the import to specifically import that instead.